### DISCUSSION TITLE
Trying to load a react app into pages but the hosted site is just a blank white screen

### DISCUSSION THREAD

**Discussion by Sulima1** [community/community#57102] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi GitHub! 

I'm trying to host my portfolio website on pages. It's a single page application built on react. I successfully made the branch for gh-pages but when I visit the pages site, it is blank. I get an output message that states 

Loading failed for the <script> with source “https://sulima1.github.io/static/js/main.d933ddca.js”.

I've been trying to find a fix for it for a while but have had no luck. I'm really new to React and web development in general so it is likely a really simple mistake but I just have no idea. 

In case you want to see the output for yourself https://sulima1.github.io/Mattborkdev/

Also the repository is public so here is the link for that https://github.com/Sulima1/Mattborkdev

Thank you!

**Comment by airtower-luna** [comment]
That error message indicates your `index.html` uses incorrect [absolute URLs](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#absolute_urls_vs._relative_urls): It tries to access a script under `/static/`, while your site is deployed at `/Mattborkdev/` (and the `static/` directory of your repository within there).

The best solution would be to use relative paths instead, if you can't do that fixing the base path should work, too.

**Reply by Sulima1** [reply]
I did implement that change but unfortunately I am still getting the Loading failed for the <script> with source “https://sulima1.github.io/static/js/main.d933ddca.js”. message.

**Reply by airtower-luna** [reply, marked as answer]
Then you didn't fix it. Your `index.html` still has the same (broken) absolute path:

```html
<script defer="defer" src="/static/js/main.d933ddca.js">
```

The path needs to be either `static/js/main.d933ddca.js` (without the leading slash), or `/Mattborkdev/static/js/main.d933ddca.js`.

**Reply by Sulima1** [reply]
Ok I added the <script> to my index.html because there was none before. It works and the website loads the plain html but not any of the css or js I implemented in the react app.

**Reply by airtower-luna** [reply]
What I quoted was already there, and the CSS and scripts have the exact same problem. You should look into how to fix the paths in the React build. I can't tell you how to do that, but probably setting some kind of base path/URL. :wink:

**Reply by Sulima1** [reply]
Thank you. I appreciate the help.
